source/css/_common/components/post/post.styl

```diff
@@ -1,11 +1,12 @@
 .post-body {
   word-wrap();
   font-family: $font-family-posts;
-  font-size: $font-size-posts;
+  //font-size: $font-size-posts;
 }
 
 .post-body span.exturl {
   .fa {
+    font-size: $font-size-base;
```

Comment:
Is there any difference? `.fa` is always `1em`, which is equal to `$font-size-base;`

Reply:
I think `.fa` must be a little smaller than text. See example with `.exturl` icon:

**A little smaller icon**:
![image](https://user-images.githubusercontent.com/16944225/56728857-16444f00-6754-11e9-8b18-f37ae9532f50.png)

**Same sized-icon**:
![image](https://user-images.githubusercontent.com/16944225/56728902-33791d80-6754-11e9-9a98-12e659065fbe.png)

U think second look pretty nice?

Reply:
Yes, I think it's fine